src/components/ViewRequest/Details.tsx

```diff
@@ -66,6 +66,15 @@ const ViewRequestDetails = ({
 
             <HiringPanel />
 
+            {request.data.currentEmployee && (
```

Comment:
Did we need to put any phase restrictions on this like was done with the Hiring Panel?  If they send it back a stage or more the data would no longer be relevant, as it is liking changing because they are selecting a new candidate?  They would be re-prompted again when moving forward so it would be corrected then.